src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -282,7 +283,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
 
   def listWorkspaces(): Future[PerRequestMessage] =
     dataSource.inTransaction { dataAccess =>
-
+      Kamon.metrics.histogram("test-histogram").record(scala.util.Random.nextInt(1000))
```

Comment:
Could you make this a real timer, instead of a random value? That way, we could use this PR as a full-featured example of how to capture metrics.